### PR TITLE
#932 updated NEWS

### LINKED ISSUE #932: News update from 0.6.0 to 0.7.0 for issue 745 examples
I merged in #745 and realized later that it is listed in 0.6.0 news release, but it actually did not make it in.  So #745 will be released into 0.7.0.

Could you update the news sections for 0.7.0 once #900 is merged into `devel `branch.  #900 is a skeleton news for the next release. 
 Sorry about that!

### PR DESCRIPTION
**Pull request by upadhyor**
Close #932 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ dataset](https://pharmaverse.github.io/admiral/articles/queries_dataset.html) re
 
 ## Documentation
 
+- Examples have been added for `format_reason_default()`, `format_eoxxstt_default()`, `extend_source_datasets()` and `filter_date_sources()` (#745)
```
Comment (bms63) [inline NEWS.md:26]: Thanks for doing this!

**Comment by bms63** [comment]
@thomas-neitmann Are we just ignoring this 3.5 error going forward other PRs are having this issue #988 #990 ?

**Comment by thomas-neitmann** [comment]
@bms63 Yes, just ignore!

File: .Rbuildignore [hunk 2]
```diff
@@ -22,3 +22,6 @@
 ^vignettes/.+png$
 ^LICENSE\.md$
 ^\.github$
+^admiral\.Rcheck$
+^admiral.*\.tar\.gz$
+^admiral.*\.tgz$
```
Comment (thomas-neitmann) [inline .Rbuildignore:27]: How did these end up here? These files are not part of the repository...

Comment (bms63) [inline .Rbuildignore:27]: Hey @upadhyor can you remove these from your branch?

**Review by bms63 — CHANGES_REQUESTED** [review]

**Review by thomas-neitmann — APPROVED** [review]
Thanks @upadhyor 👏

**Pull request merged**